test/test_parse_cmake_version.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2016-2018 Dirk Thomas
```

Comment:
Please consider use your name for the copyright of this file since you wrote it. Also the year should 2020.

Reply:
Fixed